tests/test_import_utils.py

```diff
@@ -127,9 +126,7 @@ def test_parse_sensor_data(
             email="p.ersoon@amsterdam.nl",
             telephone="06123456789",
             website="amsterdam.nl",
-            first_name="Pieter",
```

Comment:
Wat is eigenlijk de historie van deze splitsing van namen? Als dit overal terugkomt en dan weer samengevoegd wordt.